src/Controller/DefaultController.php

```diff
@@ -610,26 +610,10 @@ public function enqueueJobAction(Request $request, RequestStack $rs, $idClient,
             } else {
                 $repository = $this->getDoctrine()->getRepository('App:Job');
                 $job = $repository->findOneById($idJob);
-                if ($token == $job->getToken()) {
-                    // Valid token, but let's require HTTPS
-                    if ($rs->getCurrentRequest()->isSecure()) {
-                        $trustable = true;
-                    } else {
-                        $response = new JsonResponse(array(
-                            'status' => 'false',
-                            'msg' => $t->trans(
-                                'Aborted: HTTPS required',
-                                array(),
-                                'BinovoElkarBackup'
-                            )
-                        ));
-                        return $response;
-                    }
-                } else {
-                    // Invalid token
-                    $trustable = false;
-                }
-                
+
+                // Check token validity
```

Comment:
Do you think comment is necessary? If you think the check if a bit unclear, maybe you can consider replacing the one-liner with a if/else construct and get rid of the comment?

Reply:
you're right, I remove the comment, the next line is self documenting.